src/inverseRobot.js

```diff
@@ -7,7 +7,12 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = Object.fromEntries(Object.entries(robot)
+    .map(([key, value]) =>
+      [value, key]));
```

Comment:
The variable name 'newRobot' is not descriptive enough. It would be better to use a name that indicates that this is an inverted version of the original robot object, like 'invertedRobot'.